components/src/core/components/Input/Autocomplete/AutocompleteInput.vue

```diff
@@ -265,7 +269,7 @@ export default defineComponent({
       this.debouncer();
     },
     onBlur() {
-      if (!this.multiple && this.searchTerm) {
+      if (!this.multiple && this.searchTerm && !this.stopSearchKeyClearOnBlur) {
```

Comment:
@gayan-orangehrm Is this a requirement for the widget?  why would it need to remove the value on blur by default?

Reply:
@amilaw it was passing `null` for `update:modelValue` from the very beginning of the component. checked in the OS code base as well.
Therefore I ignore it based on newly introduced prop `stopSearchKeyClearOnBlur`

Reply:
Removed the code block which removed the value when on blur as per discussion with Chamara and some dev testings have done by myself after discussing with Chamara

Reply:
@gayan-orangehrm add the changelog.md file and add a record like below format.

`[Date] - [Commit hash] - [Affected Component] - [Remark]`

2022-03-29 -  7e7fca5 - AutocompleteInput.vue - Removed setting model value null onBlur. Possible affected components, QuickSearchInput.vue